cli/azd/pkg/input/asker.go

```diff
@@ -145,6 +145,17 @@ func askOnePrompt(p survey.Prompt, response interface{}, isTerminal bool, stdout
 		}
 		*pResponse = result
 		return nil
+	case *survey.Password:
+		var pResponse = response.(*string)
+		fmt.Fprintf(stdout, "%s", v.Message[0:len(v.Message)-1])
+		fmt.Fprintf(stdout, "%s ", v.Message[len(v.Message)-1:])
```

Comment:
Not quite understanding this change yet... Do we need two separate `printf`s here?

Reply:
it's a copy-paste effect from `case *survey.Input:` 
I just deleted the part of 

```
if v.Default != "" {
			fmt.Fprintf(stdout, " (or hit enter to use the default %s)", v.Default)
		}
```

ended up whit this printf...  Let me get rid of it.  It does bring confusion